src/components/Modal/Modal.vue

```diff
@@ -155,14 +155,14 @@
 						</Actions>
 
 						<!-- Close modal -->
-						<Actions v-if="canClose && !closeButtonContained" class="header-close">
-							<ActionButton @click="close">
-								<template #icon>
-									<Close :size="iconSize" />
-								</template>
-								{{ t('Close') }}
-							</ActionButton>
-						</Actions>
+						<ButtonVue v-if="canClose && !closeButtonContained"
```

Comment:
```suggestion
						<Button v-if="canClose && !closeButtonContained"
```
**suggestion (non-blocking):** Rename ButtonVue to Button where it's used

Reply:
It was used like this before. Don't know why.

Reply:
When there are regular `button` elements the names conflict, so we import the cue button as ButtonVue